### DISCUSSION TITLE
資安問題頻傳，蘋果 Mac 也不再安全！《TO》推薦 2022 年 Top 5 防毒軟體

### DISCUSSION THREAD

**Discussion by Rosancl** [1101-GS4719-C/Python-Community-GS4719-C#593] [General]
現在許多惡意軟體或網頁已經可以不分作業系統攻擊，因此蘋果電腦雖然還是相對安全，但是「不會中毒」的說法已經不再正確。

因此不論是 Windows 還是 MacOS 的使用者，都需要養成正確的上網觀念並搭配適當的防毒軟體，才能保護電腦資料。以下根據國外安全公司 [AV-TEST](https://www.av-test.org/en/antivirus/home-macos/) 整理了幾款防毒軟體，大家可以根據自己的預算和需求參考看看。

https://buzzorange.com/techorange/2022/01/07/mac-antivirus-software-2022/

**Comment by 11timyyy** [comment]
之前有聽說過Norton這個品牌，但有好的資安意識，才可以避免病毒的入侵。

**Comment by HsuChen88** [comment]
說到底病毒還是幫我們修電腦的人創造的
這樣電腦才可以一直拿去修

**Comment by yuhao112** [comment]
惡意軟體有些也是我們有意下載的，等此防火牆越來越堅固，下載盜版就難了....

**Comment by hapiwangy** [comment]
平常除了windows原生地和趨勢以外都不太清楚還有甚麼其他的防毒軟體，現在才知道原來還有那麼多種選擇

**Comment by JSHT** [comment]
正確的防範意識，才是最好的防毒機制。
同時，文中提到的 PC-Cillin，學校就有提供序號了，可以拿來用在家中的老電腦。

**Reply by chia-kai-chang** [reply]
原來學校有提供，我還自己花錢買PC-Cillin

**Reply by JSHT** [reply]
學校一樣在[授權軟體](https://ca.ncu.edu.tw/software/)這邊有提供

**Reply by Sazmafuji** [reply]
聽到好情報，我要快快去取得

**Comment by lx3211** [comment]
不要亂點不明連結跟網站才是真正治本的方法

**Comment by handsomeniko** [comment]
真的不要亂點連結

**Comment by bloodtrail0511** [comment]
還有一些病毒會包裝成防毒軟體，有夠恐怖

**Comment by ThomasMeng0515** [comment]
我目前用的就是學校提供的 PC-Cillin，但其實Windows原生就有內建防毒軟體可以使用，可以抵擋大部分的病毒

**Comment by Jay-722** [comment]
不要亂點就比較不會出事

**Comment by sys511613** [comment]
防毒軟體其實也蠻占用電腦資源的
如果平常不要亂點亂下載 windows內建防毒應該就很夠用了

**Comment by etin2468** [comment]
其實windows內建的就很強了，每次防毒軟體都會莫名其妙吃掉電腦一堆資源

**Comment by henrywen547** [comment]
自己本身還是要具有防毒意識啦
畢竟防毒軟體的開發者前身也大多是駭客
選擇防毒軟體除了看價錢，也要做一點研究才行

**Comment by cordydeng** [comment]
除了使用防毒軟體外，最重要的是不要亂點一些奇怪的連結才是上上策

**Comment by gnsJhenJie** [comment]
除了不要點不明連結外，確保電腦更新在最新版本也是很重要的。

**Comment by Lai27330** [comment]
除了善用防毒軟體，也要具備資安意識，不要隨便連結不明網路或是點進來歷不明的網站。

**Comment by treeann** [comment]
我的筆電友送ㄝ哈哈

**Comment by Jimmy911218** [comment]
現在網路方便很容易會被竊取個資，因此除了下載這些防毒軟體外，還要注意自己的上網模式才行

**Comment by yucheng1028** [comment]
我是都用eset

**Comment by HaoFish123** [comment]
感謝分享，防毒軟體剛好過期了來研究一下。

**Comment by stan890425** [comment]
感謝推薦

**Comment by alexthebigpp** [comment]
感謝推薦

**Comment by bloodtrail0511** [comment]
感謝分享，不然種類太多都不知道要選哪種

**Comment by Lynn12378** [comment]
大多時候會因為想看免費資源點進去一些不安全的網址

**Comment by Hchejxnc** [comment]
良好網路使用習慣，感覺才是避免中毒的重點

**Comment by williamyen0623** [comment]
基本上不要下載一些奇奇怪怪來路不明的東西, 就不會中毒

**Comment by Yunhans** [comment]
我之前在macbook上下載學校給的f-secure然後刪不乾淨

**Comment by Jasper525** [comment]
Windows defender真的很夠了
最好的還是養成定期做系統備份的好習慣
這樣中毒也不怕

**Comment by evanchou1204** [comment]
感覺防毒軟體真的會佔電腦資源，而且之前的防毒軟體還會擋vscode的執行檔，超荒謬

**Comment by Pietra1226** [comment]
防毒軟體真的很重要，而且現在也不局限在軟體端，硬體端的木馬攻擊也是很致命而且很難偵測的

**Comment by dabbie109303501** [comment]
防毒軟體蠻重要的，但也要經常更新防毒軟體的數據，才能最有效保護電腦

**Comment by CPCpyt** [comment]
防毒軟體和茲安意識真的很重要，記得我上台筆電就是因為病毒導致硬體遺失。
